internal/description/description.go

```diff
@@ -75,8 +75,8 @@ const (
 	CloudManager             = "Cloud Manager operations."
 	ShutdownCluster          = "Shutdown a cluster."
 	StartUpCluster           = "Start up a cluster."
-	ConfigDescription        = "Configure a profile. This let you store access settings to your cloud."
-	ConfigSetDescription     = "Configure specific properties of the profile."
+	ConfigDescription        = "Configure a profile. This let you store default settings."
```

Comment:
Are we sure we want to use "default settings" here? Will that cause confusion between settings stored in a default profile vs other profile? Maybe something like this:
```suggestion
	ConfigDescription        = "Configure a profile to store access settings for your MongoDB deployment."
```